AM_CONDITIONAL should be always called unconditionally, or automake
conditionals make ./configure step fail.

Move AM_CONDITIONAL for include_ALSA in global scope.

See Gentoo Bug #115379.

Index: plugins/audiooutput/alsa/configure.in.in
===================================================================
--- plugins/audiooutput/alsa/configure.in.in	(revision 488109)
+++ plugins/audiooutput/alsa/configure.in.in	(revision 488116)
@@ -55,7 +55,6 @@
   fi
 
   AC_SUBST(LIBASOUND)
-  AM_CONDITIONAL(include_ALSA, [test "x$have_alsa_0_9" = xyes || test "x$have_alsa_1" = xyes])
 ])
 
 AC_ARG_WITH(alsa,
@@ -72,4 +71,6 @@
   fi
 fi
 
+AM_CONDITIONAL(include_ALSA, [test "x$have_alsa_0_9" = xyes || test "x$have_alsa_1" = xyes])
+
 dnl --------- ALSA CHECK END ---------------
